ac-cider.el

```diff
@@ -87,7 +87,6 @@ Caches fetched documentation for the current completion call."
                   "\r" ""
                   (nrepl-dict-get (cider-nrepl-send-sync-request
                                    (list "op" "complete-doc"
-                                         "session" (nrepl-current-session)
```

Comment:
Thank you for your suggestion.  I can drop this pull request and create another with that change.  However, I'm curious.  It doesn't seem that cider-nrepl cares about the session for the complete-doc handle in complete.clj.  Is there another reason?